Tests/test_PDB_PSEA.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import unittest
+
+from Bio.PDB import PDBParser, PSEA
+
+
+os.environ['LANG'] = 'C'
+
+p-sea = None
```

Comment:
You cannot use a minus-sign/hyphen as part of a Python variable name. How about ``psea`` or ``p_sea`` here?